### DISCUSSION TITLE
Strimzi operator update

### DISCUSSION THREAD

**Discussion by Yedidyah27** [strimzi/strimzi-kafka-operator#4820] [Q&A]
Hey,
We are upgrading our strimzi operator from 0.19.0 to 0.22.1.  We are not updating the kafka cluster but we are seeing that the new operator rolling update to the kafka cluster automatically.
We are seeing that our consumer groups get disconnected from the brokers for a short period of time during this update.  Is this something that we should expect when upgrading the operator?

**Comment by scholzj** [comment]
The operator upgrade always rolls the pods - it has to make sure they are correctly configured and they are using the right images. So this is expected even when the Kafka version itself does not change.

**Reply by Yedidyah27** [reply]
Ok Thanks for the clarification
